pkg/utils/utils.go

```diff
@@ -393,3 +408,17 @@ func GetVFID(pciAddr string) (vfID int, err error) {
 	vfID = -1
 	return vfID, nil
 }
+
+// GetPfEswitchMode returns PF's eswitch mode for the given VF
+// If device is not VF then it will return its own eswitch mode
+func GetPfEswitchMode(pciAddr string) (string, error) {
+	pfAddr, err := GetPfAddr(pciAddr)
+	if err != nil {
+		return "", fmt.Errorf("error getting PF eswitch mode for PCI device %s %v", pciAddr, err)
```

Comment:
The error message is not accurate, maybe change to `error getting PF PCI address for device %s %v`.